tools/create_virtual_hardware.sh

```diff
@@ -59,7 +64,26 @@ function ensure_softnpu_zone {
     success "softnpu zone exists"
 }
 
+function warn_if_physical_link_and_no_proxy_arp {
```

Comment:
These warning seem great, but should they be a separate PR?

Reply:
Sure, they can be. I definitely ran into them while testing this, but I can split them out as well, no problem.